cmd/snap-confine/snap-confine.rst

```diff
@@ -129,9 +129,13 @@ FILES
 
 	Description of the mount profile.
 
-`/var/lib/snapd/seccomp/bpf/*`:
+`/var/lib/snapd/seccomp/bpf/*.bpf`:
 
-	Description of the seccomp profile.
+	Compiled seccomp profile program.
+
+`/var/lib/snapd/seccomp/bpf/*.in`:
+
+	Input for the /usr/lib/snapd/snap-seccomp profile compiler.
```

Comment:
You asked this elsewhere, but I'll answer here since it is clearer what is happening. Today we have text profiles in `/var/lib/snapd/seccomp/profiles`. This PR creates compiled cache files in `/var/lib/snapd/seccomp/bpf/*.bpf`. Why are we using `.in` files at all and not just using what is in `/var/lib/snapd/seccomp/profiles` as the input to `snap-seccomp`?